conf/docker/dataverse-glassfish/init-container/reference_data.sql

```diff
@@ -0,0 +1,51 @@
+-- using http://dublincore.org/schemas/xmls/qdc/dcterms.xsd because at http://dublincore.org/schemas/xmls/ it's the schema location for http://purl.org/dc/terms/ which is referenced in http://swordapp.github.io/SWORDv2-Profile/SWORDProfile.html
```

Comment:
Probably preferable to re-use an existing copy of this file (assuming one is already in the repo), rather than creating a copy that would have to be kept in sync.